src/lib/netlist/solver/nld_ms_w.h

```diff
@@ -289,43 +303,43 @@ namespace netlist::solver
 					if (H[i][i] == plib::constants<float_type>::zero())
 						plib::perrlogger("{} H singular\n", this->name());
 					const float_type f = plib::reciprocal(H[i][i]);
-					for (unsigned j = i+1; j < row_count; j++)
+					for (unsigned j = i + 1; j < row_count; j++)
 					{
-						const float_type f1 = - f * H[j][i];
+						const float_type f1 = -f * H[j][i];
 
 						// FIXME: comparison to zero
 						if (f1 != plib::constants<float_type>::zero())
 						{
-							float_type *pj = &H[j][i+1];
-							const float_type *pi = &H[i][i+1];
-							for (unsigned k = 0; k < row_count-i-1; k++)
+							float_type       *pj = &H[j][i + 1];
+							const float_type *pi = &H[i][i + 1];
+							for (unsigned k = 0; k < row_count - i - 1; k++)
 								pj[k] += f1 * pi[k];
-								//H[j][k] += f1 * H[i][k];
+							// H[j][k] += f1 * H[i][k];
```

Comment:
The commented statement logically belongs in the loop body – it makes no sense outdented like this.

Reply:
Yes - will fix this